### PR TITLE
Fix storageclass path

### PR DESCRIPTION
**Pull request by fbalak**
- Fixes https://github.com/red-hat-storage/ocs-ci/issues/193
- Storage class doesn't use `namespace`. Added default namespace so that the class can inherit from `OCS` class.

### CODE REVIEW

File: resources/storage_class.py [hunk 1]
```diff
@@ -30,22 +30,26 @@ def __init__(self, interface, **kwargs):
...
+                constants.TEMPLATE_CSI_RBD_DIR, "storageclass.yaml"
             )
             # TODO: Implement
             pass
 
         self.sc_data = yaml.safe_load(open(self.yaml_path, 'r'))
         self.sc_data.update(kwargs)
+
+        # TODO(HACK): Fix properly inheritance to OCS, storage classes don't use namespace
+        self.sc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
```
Comment (mbukatov) [inline resources/storage_class.py]: [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) objects are not in any [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/), in the same way as other low level cluster wide resources such as nodes or PVs. If the current design doesn't allow you to specify no namespace for a storage class, the design is invalid.

Comment (fbalak) [reply resources/storage_class.py]: Ok, created another issue https://github.com/red-hat-storage/ocs-ci/issues/195

**Comment by fbalak** [comment]
@RazTamir What is correct way to create storage class in current implementation (if this file will be removed)?

File: resources/storage_class.py (hunk 1 above)
Comment (nehaberry) [reply resources/storage_class.py]: > [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) objects are not in any [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/), in the same way as other low level cluster wide resources such as nodes or PVs. If the current design doesn't allow you to specify no namespace for a storage class, the design is invalid.

+1.. Storageclass should not need any namespace

**Comment by RazTamir** [comment]
@fbalak @mbukatov @nehaberry please review this - https://github.com/red-hat-storage/ocs-ci/pull/196
It should take care for any kind without namespace

**Comment by fbalak** [comment]
`namespace` argument is not passed to `StorageClass` parent now. It should work as expected after https://github.com/red-hat-storage/ocs-ci/pull/196 is merged.

@RazTamir I think that this file and `secret.py` files should be removed in different PR. I am also not sure how to correctly create resources from these files when they are deleted. `StorageClass` class gets `interface` as its required parameter so it seems that splitting sc templates into `CephFS` and `RBD` is handled by this class.

**Comment by RazTamir** [comment]
@fbalak you can use OCS in ocs.py directly like any other kind

**Comment by fbalak** [comment]
Closing as https://github.com/red-hat-storage/ocs-ci/pull/213 is merged.

**Pull request closed without merging**